manifests/enterprise.pp

```diff
@@ -0,0 +1,82 @@
+# Class splunk::enterprise
+#
+class splunk::enterprise (
+  String $version                          = $::splunk::params::version,
+  String $package_name                     = $::splunk::params::enterprise_package_name,
+  String $package_ensure                   = $::splunk::params::enterprise_package_ensure,
+  String $staging_subdir                   = $::splunk::params::staging_subdir,
+  String $path_delimiter                   = $::splunk::params::path_delimiter,
+  String $enterprise_package_src           = $::splunk::params::enterprise_package_src,
+  Optional[String] $package_provider       = $::splunk::params::package_provider,
+  Boolean $manage_package_source           = true,
+  Optional[String] $package_source         = undef,
+  Optional[Array[String]] $install_options = $::splunk::params::enterprise_install_options,
+  String $splunk_user                      = $::splunk::params::splunk_user,
+  String $enterprise_homedir               = $::splunk::params::enterprise_homedir,
+  String $enterprise_confdir               = $::splunk::params::enterprise_confdir,
+  String $service_name                     = $::splunk::params::enterprise_service,
+  String $service_file                     = $::splunk::params::enterprise_service_file,
+  Boolean $boot_start                      = $::splunk::params::boot_start,
+  Boolean $use_default_config              = true,
+  String $input_default_host               = $facts['clientcert'],
```

Comment:
`clientcert`?  That doesn't actually have to be a hostname.  Probably better to stick with `$facts['fqdn']` here.